less/navigation.less

```diff
@@ -0,0 +1,23 @@
+.container-header{
+    width: 100%;
+    .flexmix();
+    flex-direction: column;
+    background-color: @main-color;
+    padding-bottom: 10px;
+    .bottomBorder();
+
+    nav{
+        width: 100%;
+    }
```

Comment:
The navigation was responsive but not exactly centered. Look into how you might be able to do this with either display `flex` or `block`.